src/catchup/ApplyBucketsWork.cpp

```diff
@@ -26,7 +26,7 @@ ApplyBucketsWork::ApplyBucketsWork(
     Application& app,
     std::map<std::string, std::shared_ptr<Bucket>> const& buckets,
     HistoryArchiveState const& applyState, uint32_t maxProtocolVersion)
-    : BasicWork(app, "apply-buckets", RETRY_A_FEW)
```

Comment:
why no RETRY_NEVER here?

Reply:
i think you're right - I was under impression apply buckets can pick up where it left off and proceed in case there's a failure, but it looks like it doesn't even properly return `WORK_FAILURE`.